### DISCUSSION TITLE
Dependencies

### DISCUSSION THREAD

**Discussion by SolarMx6** [langflow-ai/langflow#155] [Q&A]
Hello! Tooootal newb here. But I will not be defeated!

pip install seems to have worked without error. However, when I run langflow I get this situation in Command Prompt rather than a GUI:

[23:17:01] INFO     [23:17:01] - INFO - Logger set up with log level: 20(info)                             logger.py:28
           INFO     [23:17:01] - INFO - Log file: logs\langflow.log                                        logger.py:30
←[32mINFO←[0m:     Started server process [←[36m9348←[0m]
←[32mINFO←[0m:     Waiting for application startup.
←[32mINFO←[0m:     Application startup complete.
←[32mINFO←[0m:     Uvicorn running on ←[1mhttp://127.0.0.1:7860←[0m (Press CTRL+C to quit)

Wondering if I'm missing libraries. LangChain is installed successfully as far as I can tell.

Thanks for any help!

**Comment by ogabrielluiz** [comment]
Hi!
What error exactly? 
This looks like it worked. 
All you gotta do is load that URL into your browser (or press ctrl+click the URL in most terminals).

**Comment by SolarMx6** [comment]
Thank you, Gabriel! I did not recognize the URL. I’m still fairly new to
these technical spaces, so thanks for your patience while I learn and find
my way around :)

99% of the time the answer is right in front of me lol
